frameworks/zpm.zsh

```diff
@@ -0,0 +1,52 @@
+() {
+
+local -r home_dir=${1}
+
+git clone --recursive https://github.com/zpm-zsh/zpm $home_dir/.zpm
+
+>| ${home_dir}/.zshrc <<\END
+autoload -Uz --                 \
+  git-alias-lookup              \
+  git-branch-current            \
+  git-branch-delete-interactive \
+  git-dir                       \
+  git-ignore-add                \
+  git-root                      \
+  git-stash-clear-interactive   \
+  git-stash-recover             \
+  git-submodule-move            \
+  git-submodule-remove          \
+  mkcd                          \
+  mkpw                          \
+  duration-info-precmd          \
+  duration-info-prexec          \
+  coalesce                      \
+  git-action                    \
+  git-info
+
+source ~/.zpm/zpm.zsh
+
+zpm load zimfw/environment
+zpm load zimfw/git
+zpm load zimfw/input
+zpm load zimfw/termtitle
+zpm load zimfw/utility
+zpm load zimfw/duration-info
+zpm load zimfw/git-info
+zpm load zimfw/asciiship
+zpm load zdharma/fast-syntax-highlighting
+zpm load zsh-users/zsh-completions,fpath:src
+zpm load zsh-users/zsh-autosuggestions,source:zsh-autosuggestions.zsh
```

Comment:
```suggestion
zpm load zsh-users/zsh-completions,fpath:src
zpm load zsh-users/zsh-autosuggestions,source:zsh-autosuggestions.zsh
zpm load zsh-users/zsh-syntax-highlighting,source:zsh-syntax-highlighting.zsh
```
This worked for me. It might make the comparisons fairer, as I'm not sure how faster fast-syntax-highlighting is in terms of startup time.

Reply:
Done. I've tried this earlier and it didn't work. Maybe I've mistyped something or maybe something has changed since then. Anyway, it works now.